### PR TITLE
Remove insecure wnd password example

### PR DESCRIPTION
**Pull request by IljaN**
owncloud/windows_network_drive#321

### CODE REVIEW

**Comment by IljaN** [comment]
Argh.. IDE does some formatting magic

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc [hunk 1]
```diff
@@ -281,18 +281,21 @@ The simplest way to start the `wnd:listen` process manually, perhaps for initial
 
 The password is an optional parameter and you'll be asked for it if you
 didn't provide it, as in the example above. In order to start the
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc:283]:
```suggestion
didn't provide it, as in the example above. In order to start
```

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc [hunk 2]
```diff
@@ -281,18 +281,21 @@ The simplest way to start the `wnd:listen` process manually, perhaps for initial
...
-{occ-command-example-prefix} wnd:listen <host> <share> <username> <password>
+{occ-command-example-prefix} wnd:listen <host> <share> <username> \
+  --password-file=/my/secret/password/file --password-trim
 ----
 
 For additional options to provide the password, check xref:password-options[Password Options]
 
-Note that in any case there won't be any processing of the password by default. This means that spaces or
-newline chars won't be removed unless explicitly told. Use the `--password-trim` option in those cases.
+Note, that the password must be in plain text inside the file, and neither  spaces nor newline characters will be
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc:295]:
```suggestion
Note, that the password must be in plain text inside the file, and neither spaces nor newline characters will be
```

File: modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc [hunk 3]
```diff
@@ -281,18 +281,21 @@ The simplest way to start the `wnd:listen` process manually, perhaps for initial
...
+removed from the file by default, unless the `--pasword-trim` option is added. The password file must be readable
+by the apache user (or www-data). Also make sure that the password file is outside of any directory handled
+by apache (web-readble).
```
Comment (phil-davis) [inline modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc:298]:
```suggestion
by apache (web-readable).
```

**Review by phil-davis — APPROVED** [review]
I made some small changes and squashed.

**Comment by phil-davis** [comment]
Backports:
10.6 #3239 
10.5 #3240

**Comment by IljaN** [comment]
@phil-davis Thanks!

**Pull request merged**